signer/api/api_test.go

```diff
@@ -225,3 +225,26 @@ func TestSignHandlerReturns404WithNonexistentKey(t *testing.T) {
 
 	assert.Equal(t, 404, res.StatusCode)
 }
+
+func TestCreateKeyHandlerWithInvalidAlgorithm(t *testing.T) {
+	keyStore := trustmanager.NewKeyMemoryStore(passphraseRetriever)
+	cryptoService := cryptoservice.NewCryptoService("", keyStore)
+	setup(signer.CryptoServiceIndex{data.ED25519Key: cryptoService, data.RSAKey: cryptoService, data.ECDSAKey: cryptoService})
+
+	// The `rbtree-algorithm` is expected as not suppoted
```

Comment:
Non-blocking nitpick - "supported" is mispelled.  :)  Minor though, I will merge in morning whether or not it's fixed.